Dockerfile-mongo-seed

```diff
@@ -0,0 +1,13 @@
+FROM  mongo:3.2
+
+COPY dc-campaign-finance-mongodatabase.zip .
+
+# update packages and intall unzip
+RUN apt-get update
+RUN apt-get install unzip
+
+# unzip db .bson files to root dir
+RUN unzip dc-campaign-finance-mongodatabase.zip -d .
+
+# restore mongodb using docker compose name
+CMD mongorestore --host mongodb -d dc-campaign-finance --drop ./dc-campaign-finance-mongodatabase/dc-campaign-finance
```

Comment:
if the shape of the zip changes, could the migration scripts be added then run in docker until an updated zip is available; this would rely less on a volume existing in a known state and place the data in a known state during the seed process?